scripts/kind-e2e/e2e.sh

```diff
@@ -130,6 +130,7 @@ function create_subm_vars() {
   subm_colorcodes=blue
   subm_debug=false
   subm_broker=k8s
+  subm_cabledriver=ipsec/strongswan
```

Comment:
I think we need to change this to strongswan in the end.
```suggestion
  subm_cabledriver=strongswan
```

Reply:
Or leave it blank.

Reply:
I think the `strongswan` string is better, because we can check that user can define the driver name and the name will be propagated into the `submariner` CRD, otherwise the relevant CRD filed will not exist.
What do you think ?

Reply:
Good point - agree.